iothub/device/src/Transport/Amqp/AmqpTransportHandler.cs

```diff
@@ -448,20 +431,30 @@ private void TwinMessageListener(Twin twin, string correlationId, TwinCollection
                     {
                         // This can happen if we received a message from service with correlation Id that was not set by SDK or does not exist in dictionary.
                         if (Logging.IsEnabled)
-                            Logging.Info("Could not remove correlation id to complete the task awaiter for a twin operation.", nameof(TwinMessageListener));
+                            Logging.Info("Could not remove correlation Id to complete the task awaiter for a twin operation.", nameof(TwinMessageListener));
                     }
                 }
             }
         }
 
-        protected override void Dispose(bool disposing)
+        private void RemoveOldOperations(object _)
+        {
+            _ = _twinResponseTimeouts
```

Comment:
Linq-ified.

Reply:
We could make this same change in the MQTT layer as well @tmahmood-microsoft

Reply:
I will when I get to that file.